README.md

```diff
@@ -282,7 +280,7 @@ After installing:
     --outfile=bundle.js
     ```
 
-3. Make sure you include ES6 polyfills with your bundler if you need the broadest browser support. We recommend [`@babel/polyfill`](https://babeljs.io/docs/en/babel-polyfill/).
+3. Make sure you include ES6 polyfills with your bundler if you need the broadest browser support. 
```

Comment:
We can still link devs to [babel#ployfill](https://babeljs.io/docs/en/usage#polyfill) for some guidiance.

Reply:
Widget is no longer using babel/polyfill we are using core-js directly now.